src/AspNetCoreModuleV2/CommonLib/config_utility.h

```diff
@@ -17,87 +17,33 @@ class ConfigUtility
     HRESULT
     FindHandlerVersion(IAppHostElement* pElement, STRU* strHandlerVersionValue)
     {
-        HRESULT hr = S_OK;
-        IAppHostElement                *pHandlerSettings = NULL;
-        IAppHostElementCollection      *pHandlerSettingsCollection = NULL;
-        ENUM_INDEX                      index;
-        IAppHostElement                *pHandlerVar = NULL;
+        HRESULT hr;
+        CComPtr<IAppHostElement>           pHandlerSettings = nullptr;
```

Comment:
Can you remove the double free for the tests?

Reply:
Yep